src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family as a fallback, in case the primary font isn't available. For example, you could use `font-family: Roboto, Arial, sans-serif;`.